woocommerce/woocommerce-android
# [HACK Week: Wear app] Introduce Store Order Totals complication

Summary
==========
Introduces the Order Totals complication, allowing the Wear app users to display Today's Order Totals value from their watch face.

Proper unit test coverage is being introduced in the upcoming PR: https://github.com/woocommerce/woocommerce-android/pull/12558 so this PR is marked as an unit tests exemption.

Screen Capture
==========
https://github.com/user-attachments/assets/57b8fe69-0eb0-4b2d-9c8a-6ed651315445

How to Test
==========
1. In the Wear device, select a Watch face that supports Short Text complication types
2. Go to the WearOS Watch face editing menu and select the Order Totals complication
3. Make sure the Order Totals data matches what's is displayed in the store, but with a compact format. E.g. $22000 should be displayed as $22k

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
